packages/mdc-textfield/foundation.js

```diff
@@ -420,6 +422,14 @@ class MDCTextFieldFoundation extends MDCFoundation {
       },
     });
   }
+
+  /**
+   * @param {!Array<!MutationRecord>} mutationsList 
+   * @return {!Array<string>} Returns list of attribute names.
+   */
+  getMutationAttributeNames_(mutationsList) {
```

Comment:
This should be an adapter method. We should be removing the MutationRecord from the foundation, since its environment specific.

Reply:
This method is no longer required here since this part is moved to index.js (registerValidationAttributeChangeHandler's implementation) to separate the concerns of MutationRecord logic. PTAL.